appengine/flexible/api_gateway/gateway/api_gateway.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import sys
+from flask import Flask
+app = Flask(__name__)
+
+#development ports {static: 8003, flask1: 8001, flask2: 8002}
+@app.route('/test')
+def test():
+    return "GATEWAY OPERATIONAL"
+
+@app.route('/')
+def root():
+    res = requests.get('https://static-dot-flask-algo.appspot.com/')
+    return res.content
+
+@app.route('/hello/<service>')
+def say_hello(service):
+    services = {
+        'flask1': { 'url': 'https://flask1-dot-flask-algo.appspot.com/', 'send': False },
```

Comment:
@jonparrott It doesn't seem to like the environment variable.  I'm using `os.environ['GAE_APPENGINE_HOSTNAME']`.   Getting 500's and I have now added `print(os.environ)` to see what I am working with but can't seem to find the logs.    I am getting the logged requests, but not able to see the actual error messages or logs from my `print` command.  Do you know where I would go to access this?